block-storage/src/main/scala/coop/rchain/blockstorage/finality/LastFinalizedStorage.scala

```diff
@@ -6,6 +6,10 @@ import coop.rchain.models.BlockHash.BlockHash
 trait LastFinalizedStorage[F[_]] {
   def put(blockHash: BlockHash): F[Unit]
   def get(): F[Option[BlockHash]]
+
+  /** If LFB is migrated to BlockDagStorage metadata */
```

Comment:
Question:
Do you mean `/** If LFB is migrated to BlockDagStorage metadata , we can remove this one*/`?

Reply:
It's more `Whether LFB  should be migrated to BlockDagStorage`